cli/src/summarize.rs

```diff
@@ -154,26 +154,17 @@ where
             "Job ID",
             resp.job_id.to_string(),
         ),
-        (
-            "Type",
-            ecosystem.render(),
-            "Language",
-            ecosystem.language().to_string(),
-        ),
         (
             "User ID",
             resp.user_email.to_string(),
             "View in Phylum UI",
             format!("https://app.phylum.io/projects/{}", resp.project),
         ),
     ];
-    let summary = details.iter().fold("".to_string(), |acc, x| {
-        vec![
-            acc,
-            format!("{:>16}: {:<36} {:>24}: {:<36}", x.0, x.1, x.2, x.3),
-        ]
-        .join("\n")
+    let mut summary = details.iter().fold("".to_string(), |acc, x| {
+        format!("{}\n{:>16}: {:<36} {:>24}: {:<36}", acc, x.0, x.1, x.2, x.3)
     });
+    summary = format!("{}\n       Ecosystem: {}", summary, ecosystem.render());
```

Comment:
I've decided to put this at the bottom since it's both the easiest implementation and I think it looks a bit strange to have the right side of the grid empty with other stuff below it.

If the formatting should be kept more consistent with the previous behavior and the gap in the middle is *desired*, please let me know.